packages/base/src/blockchain.ts

```diff
@@ -191,14 +205,20 @@ export const RawTransaction = table(
   ["version", "cellDeps", "headerDeps", "inputs", "outputs", "outputsData"]
 );
 
-export const Transaction = table(
+const BaseTransaction = table(
   {
     raw: RawTransaction,
     witnesses: BytesVec,
   },
   ["raw", "witnesses"]
 );
 
+export const Transaction = createBytesCodec({
+  pack: (tx: _Transaction) =>
+    BaseTransaction.pack(transformTransactionCodecType(tx)),
+  unpack: (buf) => deTransformTransactionCodecType(BaseTransaction.unpack(buf)),
+});
+
```

Comment:
any reason for `Bytes` wrapper here?

Reply:
For example, if I have an `api.Header` instance:

```ts
const header = {
  version: "0x...",
  nounce: "0x1"
}
HeaderCodec.unpack(HeaderCodec.pack(header))
//{
//  raw: {
//    version: '0x...',
//  },
//  nouce: 1,
//}
```

After the `pack` and `unpack` operation, the data changed.

Adding a test case for this: https://github.com/zhangyouxin/lumos/blob/blockchain-utils/packages/base/tests/blockchain.test.ts#L119-L126